### ISSUE TITLE
Failed to fetch Slasher address {"err": "no contract code at given address"}

### ISSUE THREAD

**Issue by bsneverend** [Layr-Labs/eigenlayer-cli#49]
Describe the bug
ERROR   logging/zap_logger.go:57        Failed to fetch Slasher address {"err": "no contract code at given address"}
github.com/Layr-Labs/eigensdk-go/logging.(*ZapLogger).Error
        /root/go/pkg/mod/github.com/!layr-!labs/eigensdk-go@v0.1.0/logging/zap_logger.go:57
github.com/Layr-Labs/eigensdk-go/chainio/utils.NewEigenlayerContractBindings
        /root/go/pkg/mod/github.com/!layr-!labs/eigensdk-go@v0.1.0/chainio/utils/bindings.go:45
github.com/Layr-Labs/eigensdk-go/chainio/clients/elcontracts.BuildELChainWriter
        /root/go/pkg/mod/github.com/!layr-!labs/eigensdk-go@v0.1.0/chainio/clients/elcontracts/writer.go:80
github.com/Layr-Labs/eigenlayer-cli/pkg.OperatorCmd.RegisterCmd.func1
        /root/go/pkg/mod/github.com/!layr-!labs/eigenlayer-cli@v0.6.0/pkg/operator/register.go:98
github.com/urfave/cli/v2.(*Command).Run
        /root/go/pkg/mod/github.com/urfave/cli/v2@v2.27.1/command.go:279
github.com/urfave/cli/v2.(*Command).Run
        /root/go/pkg/mod/github.com/urfave/cli/v2@v2.27.1/command.go:272
github.com/urfave/cli/v2.(*Command).Run
        /root/go/pkg/mod/github.com/urfave/cli/v2@v2.27.1/command.go:272
github.com/urfave/cli/v2.(*App).RunContext
        /root/go/pkg/mod/github.com/urfave/cli/v2@v2.27.1/app.go:337
github.com/urfave/cli/v2.(*App).Run
        /root/go/pkg/mod/github.com/urfave/cli/v2@v2.27.1/app.go:311
main.main
        /root/go/pkg/mod/github.com/!layr-!labs/eigenlayer-cli@v0.6.0/cmd/eigenlayer/main.go:34
runtime.main
        /usr/local/go/src/runtime/proc.go:267
no contract code at given address

To Reproduce
eigenlayer operator register operator.yaml,

rpc points to goerli testnet infura and the contract address in the operator.yaml file: I have also tried BlockPI goerli rpc

I have upgrade version to 0.6.0, everything fill like on guide,

EigenLayer Delegation Manager contract address
This will be provided by EigenLayer team
el_delegation_manager_address: 0x1b7b8F6b258f95Cf9596EabB9aa18B62940Eb0a8
chain_id: 5

Expected behavior
Please give me solution because until now it has been stopped

**Screenshoot**

<img width="960" alt="Version" src="https://github.com/Layr-Labs/eigenlayer-cli/assets/159457627/2584b3f7-2117-4fbf-bb62-972b8339382a">
Version

<img width="947" alt="operator config" src="https://github.com/Layr-Labs/eigenlayer-cli/assets/159457627/da642d08-1d9d-4375-8199-12b795740462">
operator.yaml

<img width="948" alt="register" src="https://github.com/Layr-Labs/eigenlayer-cli/assets/159457627/ca79340c-c5b1-4570-8e02-c86e360a098e">
eigenlayer operator register operator.yaml,

OS details

**Comment by Jesse-Sawa** [comment]
While the [Operator installation guide](https://docs.eigenlayer.xyz/eigenlayer/operator-guides/operator-installation) doesn't mention It, you'll need to add `el_slasher_address: 0xD11d60b669Ecf7bE10329726043B3ac07B380C22` to your operator.yaml file

When you run the command to register your operator I.e. `eigenlayer operator register operator.yaml` the file will be overwritten (thinking this might be a symptom of an issue in the cli but haven't confirmed)

**Comment by kadirchan** [comment]
same here

**Comment by shrimalmadhur** [comment]
Hey, recently updated the [docs](https://docs.eigenlayer.xyz/eigenlayer/operator-guides/operator-installation#goerli-smart-contract-addresses) to say it should be `el_delegation_manager_address` - we also had a bug in `v0.6.0` where the migration was not happening correctly. So please update to [v0.6.1](https://github.com/Layr-Labs/eigenlayer-cli/releases/tag/v0.6.1). There are some release instructions in the release page, please make sure to refer it.

basically you just need to update the field `el_delegation_manager` to `el_delegation_manager_address` to you `operator.yaml`

Hope this helps. Let me know if it doesn't

**Comment by shrimalmadhur** [comment]
@Jesse-Sawa you don't need Slasher address anymore. With updated version > `v0.6.x` - you will just need `el_delegation_manager_address`

**Comment by kadirchan** [comment]
> Hey, recently updated the [docs](https://docs.eigenlayer.xyz/eigenlayer/operator-guides/operator-installation#goerli-smart-contract-addresses) to say it should be `el_delegation_manager_address` - we also had a bug in `v0.6.0` where the migration was not happening correctly. So please update to [v0.6.1](https://github.com/Layr-Labs/eigenlayer-cli/releases/tag/v0.6.1). There are some release instructions in the release page, please make sure to refer it.
> 
> basically you just need to update the field `el_delegation_manager` to `el_delegation_manager_address` to you `operator.yaml`
> 
> Hope this helps. Let me know if it doesn't

okay see that thanks

**Comment by shrimalmadhur** [comment]
I am assuming this is resolved so closing it. please feel free to re-open if there are any issues.

**Comment by ssachin520280** [comment]
It is el_delegation_manager_address

Even then it is showing error in cli v0.11.2
